src/components/Header/HeaderControls.tsx

```diff
@@ -110,7 +110,12 @@ export function HeaderControls({ setIsCodeFlyoutVisible }: IHeaderControls) {
           </ControlButton>
         </EuiFlexItem>
       )}
-      <EuiFlexItem grow={recordingStatus === RecordingStatus.Recording}>
+      <EuiFlexItem
+        grow={
+          recordingStatus === RecordingStatus.Recording ||
+          recordingStatus === RecordingStatus.Paused
+        }
+      >
```

Comment:
Would this work? (I've tested locally but not sure if I missed some reasons behind it)
```suggestion
      <EuiFlexItem grow>
```